docs/API.md

```diff
@@ -946,15 +960,44 @@ pre-1.3.0 version is available only as** `SRTO_LATENCY`.
 | --------------------- | ----- | ------- | ------ | ------ | ------- | ------ |
 | `SRTO_RCVSYN`         |       | pre     | `bool` | true   | true    | false  |
 
-- **[GET or SET]** - Synchronous (blocking) receive mode 
+- **[GET or SET]** - When true, sets blocking mode on reading function, when
+it's not ready to perform the operation. When false ("non-blocking mode"), the
+reading function will in this case report error `SRT_EASYNCRCV` and return
+immediately. Details depend on the tested entity:
+
+- On a connected socket or group this applies to a receiving function
+(`srt_recv` and others) and a situation when there are no data available for
+reading. The readiness state for this operation can be tested by checking the
+`SRT_EPOLL_IN` flag on the aforementioned socket or group.
+
+- On a freshly created socket or group that is about to be connected to a peer
+listener this applies to `srt_connect` call (and derived), which in
+"non-blocking mode" always returns immediately. The connected state for that
+socket or group can be tested by checking the `SRT_EPOLL_OUT` flag. NOTE
+that a socket that failed to connect doesn't change the `SRTS_CONNECTING`
+state and can be found out only by testing `SRT_EPOLL_ERR` flag.
+
+- On a listener socket this applies to `srt_accept` call. The readiness state
+for this operation can be tested by checking the `SRT_EPOLL_IN` flag on
+this listener socket. This flag is also derived from the listener socket
+by the accepted socket or group, although the meaning of this flag is
+effectively different.
+
+- Note that this flag set on a group is for group only, as it concerns the
+exact receiving operation being done on a group.
```

Comment:
- Note that this flag is for a group only, as it concerns a
specific receiving operation being done on a group.

Reply:
No, that's not what I meant. This flag can be set on a socket or on a group - not only on a group. The difference is that whereas most of the options can be set on the group only to be derived by the member socket (and are therefore meaningless for the group itself), this option when set on a group, is exactly meaningful for the group, and not derived by the member socket.

Reply:
How about this:
"Note that when this flag is set only on a group, it applies to a
specific receiving operation being done on that group (i.e. it is not 
derived from the socket of which the group is a member)."

Reply:
@ethouris If you are okay with the change above, I will apply it to the other similar instances below.

Reply:
Yes, that's very good.